jetty-home/src/main/resources/modules/jolokia.mod

```diff
@@ -15,9 +15,12 @@ jmx
 etc/jolokia.xml
 
 [files]
-maven://org.jolokia/jolokia-war/1.3.3/war|lib/jolokia/jolokia.war
+maven://org.jolokia/jolokia-war/${jolokia.version}/war|lib/jolokia/jolokia.war
```

Comment:
as explained in the PR description ;) 
``` 
to prevent issues with the current usage of ${} in mod files. 
Interpolation of values from pom will be based on the delimiter @....@ such @log4j2.version@
```
this file have the following section 
```
[ini]
jolokia.version?=@jolokia.version@
```
this section will be transformed to 
```
[ini]
jolokia.version?=3.5.4 (or whatever the value coming from the pom)
```
The idea is to keep the the option to run jetty using such parameter to test new jolokia version 
```
java -jar start.jar jolokia.version=6.4.4
```

Reply:
To me the use of `@blah@` is processed by the resource plugin filtering resulting in a "hardcoded" value at build time.
The use of `${blah}` is left alone.

Reply:
yes I didn't want to touch anything about `${bla}` because already used in mod files.

Reply:
I don't think this difference is documented anywhere? So you're saying that if the substitution happens at runtime, like when the start mechanism processes the .mod/.ini file then you use `${var}`, but if it happens at build time, then you use `@var@`??? That's a pretty subtle difference, think that needs a good explanation somewhere.

Reply:
@janbartel this (the `@blah@` to value at compile time) is standard Maven Resource handling with filtering.

The `@var@` syntax is compile time (expanded during the compile/package steps)
The `${var}` syntax is runtime. (expanded during runtime execution, and in the specific cases on this PR, only by jetty-start)

Reply:
Btw, we use the same `@var@` syntax in our `jetty-maven-plugin/src/it/*` and `jetty-jspc-maven-plugin/src/it/*` test cases.
Also, the `@var@` syntax is the most common form of maven resource filtering (during the various resource phases of the build).